src/8890_patches/8895_specific/frameworks_native/0001-SufraceFlinger-add-property-for-disabling-hwc.patch

```diff
@@ -0,0 +1,29 @@
+From 5e56c8e88bcd39bc045e1ebd997f15e148fa00a6 Mon Sep 17 00:00:00 2001
```

Comment:
8895_specific dir and its contents should be removed. See current directory structure